Makefile

```diff
@@ -453,6 +453,10 @@ $(HOME)/go/bin/swagger:
 api/openapi-spec/swagger.json: $(HOME)/go/bin/swagger $(SWAGGER_FILES) dist/MANIFESTS_VERSION hack/swaggify.sh
 	swagger mixin -c 412 $(SWAGGER_FILES) | sed 's/VERSION/$(MANIFESTS_VERSION)/' | ./hack/swaggify.sh > api/openapi-spec/swagger.json
 
+.PHONY: docs
+docs: api/openapi-spec/swagger.json
```

Comment:
I'm not sure I consider this file documentation. It's part of the public API really.